pyment/docstring.py

```diff
@@ -1990,7 +1992,8 @@ def _set_raw_params(self, sep):
             )
             if len(self.docs['out']['params']):
                 for p in self.docs['out']['params']:
-                    raw += self.docs['out']['spaces'] + self.dst.get_key('param', 'out') + ' ' + p[0] + sep + with_space(p[1]).strip()
+                    description = sep + with_space(p[1]).strip() if p[1] else ''
```

Comment:
It is not quite clear to me what this line does, would you, if you consider it appropriate, consider including a comment on what it does?